configure-windows-visual-studio.bat

```diff
@@ -261,8 +272,41 @@ set /p PTHREAD_HOME=    Please specify pthreads-win32 path: %=%
 :: if input dir doesn't exist print an error to the user and go back to previous line
 if not exist "%PTHREAD_HOME%"\ (echo Directory does not exist!! && goto while2)
 
+echo.
+echo.
+
+:: ask user about using zstd
+echo ZStd compression support may be added when compiling PcapPlusPlus.
+echo For downloading ZStd SDK (developer's pack) please go to https://github.com/facebook/zstd/releases
+:while3
+set /p USE_ZSTD=     To enable ZStd support type use_zstd otherwise type no: %=%
+if "%USE_ZSTD%" NEQ "use_zstd" if "%USE_ZSTD%" NEQ "no" (echo Please choose one of "use_zstd" or "no" && goto while3)
+
+
+if "%USE_ZSTD%" EQU "use_zstd" (
+::Can't use set staement inside an if so we have to do a little trickery here
+goto while4
+)
+goto skipWhile4
```

Comment:
instead of having this, maybe we can just ask for zstd SDK path and let the user know they can type "no" or simply press "return" if they want to skip it

Reply:
Done